src/desktop/components/main_layout/templates/search_results.jade

```diff
@@ -0,0 +1,8 @@
+extends react_redesign
```

Comment:
#minor nitpick incoming:

I'm not sure that this needs an entire layout here, you can probably update `react_redesign` to have a:

```
if loading
  #loading-container
    != loadingComponent
```

in there after the `#react-root`, and continue to pass a `loadingBlock`.

_or_

I would go with a template like https://github.com/artsy/force/blob/72687e28e1d78755376537abad94b6479ea8ccc2/src/desktop/apps/search2/templates/index.jade defined in the app, so it doesn't need to be in this more general desktop components folder and is better co-located with the relevant code.

This commit https://github.com/artsy/force/pull/3895/commits/72687e28e1d78755376537abad94b6479ea8ccc2 is an example of that.

Either of those (my pref is towards the latter), feels better than adding a whole new layout in this folder.